thr/urls.py

```diff
@@ -31,8 +31,15 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 
+from django.contrib import admin
+from django.urls import path, include
+
+
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('', include('thr_web.urls')),
     path('', include('users.urls')),
+
+    # REST Framework URLs
+    path('api/v1/', include('users.api.urls'), name='thr_api'),
```

Comment:
I thought API versioning is a good practice, in case we want to introduce some breaking changes in the future.
Can you please elaborate on why it's not a good idea?

Reply:
You have to support it for ever (kinda) and that can be difficult if there are big changes. The further you diverge the more difficult the backward compatibility. You can do it if you want, since the interface you're coding is already "legacy". On the other hand, the ways that one interacts with submitted hubs are limited.